backend/src/main/kotlin/dev/dres/DRES.kt

```diff
@@ -29,6 +30,10 @@ import kotlinx.dnq.util.initMetaData
 import java.io.File
 import java.nio.file.Path
 import java.nio.file.Paths
+import kotlin.io.path.exists
+import kotlin.io.path.isDirectory
+import kotlin.io.path.isExecutable
+import kotlin.io.path.listDirectoryEntries
```

Comment:
what are these for?

Reply:
In the first version, ffmpeg checks were implemented here, I removed them.